app/controllers/businesses_controller.rb

```diff
@@ -10,12 +10,15 @@ def create
     @business = current_user.build_business
     @business.assign_attributes(permitted_attributes(@business))
 
-    if @business.save
-      url = stored_location_for(:user) || developers_path
-      event = Analytics::Event.added_business_profile(url)
-      redirect_to event, notice: t(".created")
-    else
-      render :new, status: :unprocessable_entity
+    respond_to do |format|
+      if @business.save
+        url = stored_location_for(:user) || developers_path
+        Analytics::Event.added_business_profile(url)
+        format.html { redirect_to stripe_checkout_path }
+        format.turbo_stream
+      else
+        format.html { render :new, status: :unprocessable_entity }
```

Comment:
What happens for an invalid form via Turbo Stream?

Reply:
It will be handled gracefully by the redirect since it has the status of unprocessable_entity. I tried it with disabled javascript and with failed turbo. Please see the screencast shared in the PR.

Reply:
Tried it in multiple browsers, all working.

Reply:
Tried it in multiple browsers, all working.